recipes/minuit2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "minuit2" %}
+{% set version = "6.19.01" %}
```

Comment:
Tomorrow I’ll ask the ROOT devs if this can be versioned independently as tying it to the ROOT version doesn’t really make sense

Reply:
@brianv0 How do you use (or plan to use) minuit2?

The stand-alone build system might be removed in favour of only supporting minuit2 as part of a full ROOT installation (which is already available from conda-forge).

Reply:
Correction: *This PR* is causing the ROOT team to consider backtracking on supporting the standalone, since it collides with ROOT and that is not supported by the ROOT team. The standalone version of Minuit2 is very useful as an integrated part of other packages (GooFit, iMinuit, etc), but was not intended to be placed somewhere in contention with ROOT itself.

Reply:
My current recommendation is to make sure users can't install Minuit2 if they have ROOT (since it's included there)

Reply:
That's fine until another package depends on minuit2. Is it supposed to have two builds, one depending on `root` and one depending on `minuit2`?

This is already starting to happen with `cling` where conda-forge now has three conflicting builds of it (`cling`, `root` and `cppyy`).

Reply:
The one depending on ROOT could just be a do-nothing build that just adds ROOT as a dependency.

Reply:
In our particular use case, we use minuit2 in one specific instance of an image differencing algorithm.

We have an older version of Minuit2 in a separate conda channel, but we are moving to relying on conda-forge for our third party dependencies in an effort to speed up and simplify development efforts as well as the build and installation of our software by astronomers, and including the entire ROOT package would be a bit antithetical to that goal.

We've discussed rewriting that C++ code to be based on scipy minimization algorithms, but my personal responsibilities lie with packaging at the moment.

I'm happy with making minuit2 and root mutually exclusive, I wasn't able to quickly ascertain a method for doing that. Do I need to actually create two recipes?

e.g. if you rely on minuit2, you can either get:

`minuit2 * minuit2_standalone`
`minuit2 * root`

Reply:
That is a good suggestion 👍

I've implemented it using variants and multiple outputs, does this look reasonable to you? If you want to, you can build a specific variant locally using:
```bash
conda build recipes/minuit2 --variants '{"minuit2_variant": "minuit2_standalone"}'
```